tes-adapter/src/main/java/com/epam/pipeline/Application.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.pipeline;
```

Comment:
please, for the base package use com.epam.pipeline.tesadapter in according to our hierarchical scheme